sdk/identity/identity/src/msal/nodeFlows/msalClientCertificate.ts

```diff
@@ -132,7 +132,8 @@ export class MsalClientCertificate extends MsalNode {
         scopes,
         correlationId: options.correlationId,
         azureRegion: this.azureRegion,
-        authority: options.authority
+        authority: options.authority,
```

Comment:
Should this be
```suggestion
        authority: options?.authority,
```

Reply:
oh, actually: the claims line should just be `options.claims` — options will default to `{}`, so it will always be present.

Reply:
I’ll make these consistent (per file basis), one moment